docs/guides/how_to_guides/configuring_data_contexts/how_to_instantiate_a_data_context_on_an_databricks_spark_cluster.rst

```diff
@@ -0,0 +1,167 @@
+.. _how_to_instantiate_a_data_context_on_a_databricks_spark_cluster:
+
+How to instantiate a Data Context on Databricks Spark cluster
+=========================================================
+
+This guide will help you instantiate a Data Context on an Databricks Spark cluster.
+
+The guide demonstrates the recommended path for instantiating a Data Context without a full configuration directory and without using the Great Expectations :ref:`command line interface (CLI) <command_line>`.
+
+.. admonition:: Prerequisites: This how-to guide assumes you have already:
+
+    - :ref:`Followed the Getting Started tutorial and have a basic familiarity with the Great Expectations configuration<getting_started>`.
+
+Steps
+-----
+
+This how-to-guide assumes that you are using the Databricks File Store (DBFS) as the Datasource, Metadata Store, and DataDocs store. The `DBFS is a file store <https://docs.databricks.com/data/databricks-file-system.html>`_ that is native to Databricks clusters and Notebooks. Files on DBFS can be written and read as if they were on a local filesystem, just by adding the `/dbfs/FileStore/` prefix to the path. For information on how to configure Databricks for filesystems on Azure and AWS, please see the associated documentation in the Additional Notes section below.
```

Comment:
This is the section that I need feedback on...

The dilemma I am facing is this : 

1) DBFS is very easy to use if you are accessing it from a Databricks notebook, and works very much in the same way that a `TupleFilesystemStoreBackend` does.  
2) But this is a different workflow from what is described in the current documentation which ( I think ) assumes that we are interacting with GE via the CLI

The question I have is : how should the documentation for datasource / metadata store and datadocs store be handled in this case?

Reply:
remove reference to 'Datasource' from DBFS